src/main.c

```diff
@@ -191,12 +193,14 @@ init_signals(void)
 		exit(1);
 	}
 
-	/* Trap SIGPIPE */
+	// Trap SIGPIPE
+
 	/* This is done so that when libhttpd does a socket operation on
 	 * a disconnected socket (i.e.: Broken Pipes) we catch the signal
 	 * and do nothing. The alternative is to exit. SIGPIPE are harmless
 	 * if not desirable.
 	 */
+
 	debug(LOG_DEBUG, "Setting SIGPIPE  handler to SIG_IGN");
```

Comment:
an extra line sneaked in here :)

Reply:
It was deliberate ;-)
As it is an inline comment, the extra line adds to visual balance and readability in my opinion.